articles/migrations/index.md

```diff
@@ -25,6 +25,102 @@ If you need help with the migration, create a ticket in our [Support Center](${e
 
 Current migrations are listed below, newest first. For migrations that have already been enabled see [Past Migrations](#past-migrations).
 
+### Summary of Endpoint Migrations
+
+This table is a summary of the below endpoint migrations for the first part of 2018. See the entries below the table for more detailed explanations.
+
+| Endpoint | Replacement | Deadline |
+| --- | --- | --- | --- |
+| usernamepassword/login | /co/authenticate | April 01, 2018 |
+| [/oauth/ro](/api/authentication#resource-owner) | [/oauth/token](/api/authentication#authorization-code) | July 01, 2018 |
+| /ssoData | /authorize?prompt=none | April 01, 2018 |
+| [/tokeninfo](/api/authentication/reference#get-token-info) | [userinfo](/api/authentication#get-user-info) | June 01, 2018 |
+| [/delegation](/api/authentication#delegation) | TBD | June 01, 2018 |
+| [/oauth/access_token](/api/authentication#social-with-provider-s-access-token) | TBD | TBD |
+
+### Introducing Lock v11, Auth0.js v9
+
+| Severity | Grace Period Start | Mandatory Opt-In|
+| --- | --- | --- |
+| Medium | 2017-12-21 |  2018-04-01 |
+
+We’re continually improving the security of our service. As part of this, we are deprecating a set of APIs (`usernamepassword/login`, `/ssodata`, [tokeninfo](/api/authentication/reference#get-token-info), [/delegation](/api/authentication#delegation)) used by Lock.js v8, v9, and v10 and and auth0.js, v6, v7, and v8. You should update your applications by **April 1, 2018**.
+
+We are offering two options for this update:
+
+1. We are moving towards a centralized login experience and are recommending this option moving forward. There are many benefits of using a centralized login experience, [read more about how Centralized Login works](/hosted-pages/login). Moving to the centralized login experience is straightforward and will simplify your code base. Read [this guide comparing centralized and embedded login](/guides/login/centralized-vs-embedded) to learn more about the differences between the two. If you choose to use centralized login, you will continue to use Lock 10 or Auth0.js v8 inside your Auth0 login page, as the new versions are designed for embedded scenarios.
+1. If you decide you need to continue using embedded login, here are the migration guides for [Auth0.js](/libraries/auth0js/v9/migration-guide) and [Lock](/libraries/lock/v11/migration-guide).
+
+#### Am I affected by the change?
+
+If you are currently using Lock v9 or v10, or Auth0.js v7 or v8, you will be affected.
```

Comment:
Shouldn't this be `If you are currently using Lock v9 or v10, or Auth0.js v7 or v8 with embedded login? I'm not affected if I'm using them with HLP.